hedera-node/hedera-config/src/main/java/com/hedera/node/config/sources/DynamicConfigSource.java

```diff
@@ -54,6 +55,17 @@ public String getValue(@NonNull final String s) throws NoSuchElementException {
         return properties.getProperty(s);
     }
 
+    @Override
+    public boolean isListProperty(@NonNull final String propertyName) throws NoSuchElementException {
+        return false;
```

Comment:
We should create an issue to support lists in DynamicConfigSource

Reply:
https://github.com/hashgraph/hedera-services/issues/17101